proto/viam/app/v1/app.proto

```diff
@@ -604,9 +604,10 @@ message GetRobotPartResponse {
 
 message GetRobotPartLogsRequest {
   string id = 1;
-  bool errors_only = 2;
+  bool errors_only = 2 [deprecated = true];
```

Comment:
Should we include a comment pointing to the ticket to remove this down the road?

Reply:
Added the comment